content/en/blog/_posts/2018-12-05-kubeadm-ga-release.md

```diff
@@ -0,0 +1,91 @@
+ * **Stable underlying implementation** — kubeadm now creates a new Kubernetes cluster using methods that shouldn't change any time soon. The control plane, for example, is run as a set of static Pods, bootstrap tokens are used for the **kubeadm join** flow, and [ComponentConfig](https://github.com/kubernetes/community/blob/master/keps/sig-cluster-lifecycle/0014-20180707-componentconfig-api-types-to-staging.md) is used for configuring the kubelet.
+ * **Configuration file schema** — With the new  **v1beta1** API version, you can now tune almost every part of the cluster declaratively and thus build a "GitOps" flow around kubeadm clusters. In future versions, we plan to graduate the API to version **v1** with minimal changes (and perhaps none).
+ * **The "toolbox" interface of kubeadm** — Also known as **phases**. If you don't want to consume the all of the **kubeadm init** tasks performed, you can perform fine-grained, generic actions with the **kubeadm init phase** command (for example, generating certificates or control plane Static Pod manifests).
+ * **Upgrades between minor versions** — The **kubeadm upgrade** command is now fully GA. It handles control plane upgrades for you, which includes upgrades to etcd, the API Server, the Controller Manager, and the Scheduler. You can seamlessly upgrade your cluster between minor or patch versions (e.g. v1.12.2 -> v1.13.1 or v1.13.1 -> v1.13.3)
+ * **etcd setup** — etcd is now set up in a way that both is secure by default, with TLS communication everywhere, and allows for expanding to a highly available cluster when needed
+
+### Who will benefit from a stable kubeadm
+
+SIG Cluster Lifecycle has identified a handful of likely kubeadm user profiles, although we expect that kubeadm at GA can satisfy many other scenarios as well.
+
+Here's our list:
+
+- You're a **new user** who wants to take Kubernetes for a spin. kubeadm is the fastest way to get up and running on [Linux machines](https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/). If you're using [Minikube](https://github.com/kubernetes/minikube) on a Mac or Windows workstation, you're actually already running kubeadm inside the Minikube VM!
+- You're a **system administrator** responsible for setting up Kubernetes on bare metal machines and you want to quickly create Kubernetes clusters that are secure and in conformance with best practices and also configurable.
+- You're a **cloud provider** who wants to add a Kubernetes offering to your suite of cloud services. kubeadm is the go-to tool for creating clusters at a low level.
+- You're an **organization with special needs**. Existing public cloud offerings like [Amazon EKS](https://aws.amazon.com/eks/) and [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/) won't cut it for you; you need customized Kubernetes clusters tailored to your hardware, security, policy, and other needs.
```

Comment:
This point is counter to the others on the list. (kubeadm is not suitable)

Reply:
why not? kubeadm is granular enough to be integrated in a 'custom installer' that calls on other APIs, like openstack, vmware, public clouds... et cetera

Reply:
nak, the flow makes sense and is correct on re-read. 

Do you have a proposed alternative phrasing?